src/Lender.sol

```diff
@@ -36,6 +36,8 @@ contract Lender {
     bool public halted;
     /// @notice address on which ETH swaps are conducted to purchase LSTs
     address public ETHWrapper;
```

Comment:
Maybe we could have an authorized setter for the ETHWrapper, now that it's an external contract with an interface? This way we could update the ETHWrapper if there were ever any issues?